pdbApp/pdbgroup.cpp

```diff
@@ -350,8 +360,10 @@ void PDBGroupPut::put(pvd::PVStructure::shared_pointer const & value,
         putpvif[i].reset(info.builder->attach(info.chan, value, info.attachment));
     }
 
-    pvd::Status ret;
-    if(atomic) {
+    if(!ret.isOK()) {
```

Comment:
Why check reg.isOK here?   Nothing has touched ret since it's default constructor set it OK.

Reply:
I think this is a leftover for an earlier iteration where more happened in the loop above.  I'll remove it.